lib/uppy/ping/scheme.ex

```diff
@@ -0,0 +1,17 @@
+defmodule Uppy.Ping do
+  use Ecto.Schema
+
+  alias Uppy.Job
+
+  @primary_key {:id, :binary_id, autogenerate: true}
+  @foreign_key_type :binary_id
+
+  schema "pings" do
+    field :response, :string
```

Comment:
Do we need to save the whole response or could this be just a statuc code int?

Reply:
[4f26776](https://github.com/andreicek/uppy/pull/2/commits/4f26776e08e2c02f021be8ec320c09cc18d0ac88)